pkg/operator/starter.go

```diff
@@ -190,6 +252,9 @@ var RevisionSecrets = []revision.RevisionResource{
 	// this is needed so that the cert syncer itself can request certs.  It uses localhost
 	{Name: "kube-apiserver-cert-syncer-client-cert-key"},
 	{Name: "kubelet-client"},
+
+	// etcd encryption
+	{Name: "encryption-config", Optional: true},
```

Comment:
Does optional mean that a Secret read error in the kubelet, some race or some admin deleting the secret by accident will lead to an API server to start with invalid encryption config? This will stop it from reading data, but even worse: it might make it store data unencrypted. CVE ahead.

Reply:
Need to think on this some.